File: atividade_py/atividade03.py
faturamento = 0.0;

for i in range(1, 6):
    
    print(f"Informe o nome do {i}º Cliente: ", end="");
    nome = input();
    print(f"Informe o valor gasto por {nome}: ", end="");
    valor_pago = float(input());

    faturamento = faturamento + valor_pago;

print(f"Faturamento {faturamento}R$ total da Loja");

if faturamento > 54000:

    total = faturamento - 54000;
    print(f"Superamos em {total}R$ faturamento da Loja B");
else:
    total = 54000 - faturamento;
    print(f"Faltou {total}R$ para superarmos o faturamento da Loja B");

File: 2024/atividade_C/atividade03.c
#include <stdio.h>

char nome[30];
float valor_pago, faturamento,total;

int main() {

    faturamento = 0;
    for(int i=1; i<=5 ; i++)
    {
        printf("Informe o nome do %d° Cliente: ",i);
        scanf("%s", &nome);
        printf("\nInforme o Valor gasto pelo %d° Cliente: \n",i);
        scanf("%f", &valor_pago);

        faturamento = faturamento + valor_pago;
    }

    if(faturamento > 54000)
    {
        total = faturamento - 54000;
        printf("Superamos em %.2fR$ faturamento da Loja B\n", total);
    }

    printf("Faturamento Total: %.2fR$",faturamento);

    return 0;
}